HW2/P5/P5.txt

```diff
@@ -0,0 +1,23 @@
+
+MULTITHREADING
```

Comment:
Very detailed result! Great work!
Part 3 (sorting) accelerated my algorithm... maybe it is due to different machine performances. It is serial but only happens once for hundreds of iteration. And it may accelerate the algorithm by reducing the time accessing data from memory.